### PR TITLE
feat(chat-e2e): updated tests with models grouped by version feature

### PR DESCRIPTION
**Pull request by irinakartun**
**Description:**

- updated tests with models grouped by version feature
- automated test-case EPMRTC-1945

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix:`, `feat:`, `feature:`, `chore:`, `hotfix:` or `e2e:`
- [x] if contains breaking changes then the pull request name starts with `fix!:`, `feat!:`, `feature!:`, `chore!:`, `hotfix!:` or `e2e!:`
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-851.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-851.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by dspashynskyi** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-851.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by dspashynskyi** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-851.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by dspashynskyi** [comment]
/deploy-review
>- **Deployment failed**: [canceled](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/450211)

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-851.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-851.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-851.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/.env.development [hunk 1]
```diff
@@ -85,4 +85,4 @@ FOOTER_HTML_MESSAGE="For API access please fill&nbsp;<a href='#requestApiKey'><u
 CODE_GENERATION_WARNING="Full responsibility for code correctness, security and licensing lies solely with the user, not with DIAL platform or LLM vendor."
 
 ## Text for announcement banner
-ANNOUNCEMENT_HTML_MESSAGE="Welcome to AI Dial Chat application!"
+ANNOUNCEMENT_HTML_MESSAGE='Welcome to <a href="about:blank" target="_blank">AI Dial</a>! Unified AI Access for Enterprises. Secure, scalable and customizable enterprise-grade AI ecosystem that seamlessly integrates with your data and workflows, tailored to achieve your unique business objectives.'
```
Comment (mikitabut) [inline apps/chat/.env.development:88]: Why are you changed this?

Comment (irinakartun) [reply apps/chat/.env.development:88]: to make banner message consistant with message configured on preview/e2e env. Different banner message causes tests failure during local run

**Review by mikitabut — APPROVED** [review]

**Pull request merged**